packages/vertexai/package.json

```diff
@@ -39,6 +39,7 @@
     "test:ci": "yarn testsetup && node ../../scripts/run_tests_in_ci.js -s test",
     "test:skip-clone": "karma start",
     "test:browser": "yarn testsetup && karma start",
+    "test:integration": "karma start --integration",
```

Comment:
You'll have to call this from "test" (line 38) if you want to run it on PRs or do you not want to do that yet?

Reply:
I would prefer not to add this to `test` since then it'll be ran when someone runs `test` in the root directory- which is going to fail if someone hasn't defined the config that's used in these tests. Could be annoying for people working on other SDKs